scheduler/requirements.txt

```diff
@@ -3,3 +3,4 @@ celery==4.4.2
 django-celery-beat==2.0.0
 dj-database-url==0.5.0
 psycopg2-binary==2.8.5
+redis==3.5.1
```

Comment:
This shouldn't be necessary. We were using Redis as the broker but have moved to RabbitMQ. The 

`BROKER_URL=redis://localhost:6379/0 \` 

line in the `Makefile` should be changed to

`BROKER_URL="amqp://stencila:password@localhost:5672/stencila" \` 

rather than adding a dependency here.